torch/csrc/distributed/c10d/quantization.cpp

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
Can we group the cpp/cu related files in `torch/csrc/distributed/c10d/quantization` folder instead?